code/game/objects/items/grenades/plastic.dm

```diff
@@ -125,3 +125,16 @@
 	item_state = "plasticx4"
 	directional = TRUE
 	boom_sizes = list(0, 2, 5)
+
+
+// x-com ufo defense high ex charge 1993
+/obj/item/grenade/c4/satchel_charge
```

Comment:
This thing just feels a bit cheap to me. It pretty much only rewards knowing whats in the ruin beforehand, or having been there before.

Reply:
I'm not entirely sure how to rework it while still keeping the intent of it being a gotcha trap (that admittedly, won't trigger too often)